components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -527,7 +528,8 @@ public static NativeObject jsFunction_mapExistingOauthClient(Context cx, Scripta
                 String applicationName = (String) apiData.get("applicationName", apiData);
 
                 String keyType = (String) apiData.get("keytype", apiData);
-                Map<String, Object> keyDetails = getAPIConsumer(thisObj).mapExistingOAuthClient(jsonString, userName, clientId, applicationName, keyType);
+                String tokenType = "DEFAULT";
```

Comment:
In line 137 "DEFAULT_TOKEN_TYPE is already defined. We can remove this and use that instead.